### PR TITLE
Document `GET /source/{project_name}/{package_name}/_attribute/{attribute_name}` endpoint

### PR DESCRIPTION
**Pull request by saraycp**
With the attribute name specified in the parameters, this endpoint displays the attribute of the package.

**Testing Tips**

- Go to Attributes tab in the package tab
- Add one attribute
`curl -u Admin:opensuse -H 'Content-Type: application/xml; charset=utf-8' -X  GET 'http://localhost:3000/source/home:Admin/ad/_attribute/OBS:MakeOriginOlder'`

### CODE REVIEW

File: src/api/public/apidocs-new/paths/source_project_name_package_name_attribute_attribute_name.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      description: |
+        If the attribute doesn't contain any value and `with_default` is present,
+        the default values will be displayed, if any.
+      example: 1
+    - name: with_project
+      in: query
+      schema:
+        type: string
+      description: |
+        Passing `with_project`, the response displays the specied attribute of the package's project, if any, in addition to the package one.
```
Comment (eduardoj) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_attribute_attribute_name.yaml:39]:
```suggestion
        Passing `with_project`, the response displays the specified attribute of the package's project, if any, in addition to the package one.
```
Typo.

File: src/api/public/apidocs-new/paths/source_project_name_package_name_attribute_attribute_name.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      description: Wrong value for deleted parameter.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            invalid_attribute:
+              value:
+                code: 400
+                summary: "Attribute 'OwnerRootProjectTest' must be in the $NAMESPACE:$NAME style"
```
Comment (eduardoj) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_attribute_attribute_name.yaml:72]:
```suggestion
      description: |
        Bad Request.

        XML Schema used for body validation: [status.xsd](../schema/status.xsd).
      content:
        application/xml; charset=utf-8:
          schema:
            $ref: '../components/schemas/api_response.yaml'
          examples:
            invalid_attribute:
              value:
                code: 400
                summary: "Attribute 'OwnerRootProjectTest' must be in the $NAMESPACE:$NAME style"
              description: Wrong value for deleted parameter.
```
We should add a general description for all 400 errors, and a particular description for each example.

Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_attribute_attribute_name.yaml:72]: Hm... wait. This particular description doesn't fit in that example. :thinking:

Comment (saraycp) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_attribute_attribute_name.yaml]: I realized while addressing what you mentioned before, I updated. Please have another look

Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_attribute_attribute_name.yaml]: Fine now. :+1:

**Review by eduardoj — APPROVED** [review]

**Pull request merged**